index.md

```diff
@@ -1,51 +1,61 @@
-# Health Record Ingestion service
+# Introduction
 
-The HRI is a deployment ready service for streaming Health-related data into the cloud. It provides a "front door" for "Data Integrators" to send data into the IBM Cloud, while supporting both batch-processing and data streaming workflows. It provides features to initiate and track the movement of a dataset into the IBM cloud for both ["Data Integrators"](glossary.md#data-integrator) and ["Data Consumers"](glossary.md#data-consumer). 
+**[Health Record Ingestion](glossary.md#health-record-ingestion)** is a deployment-ready service for streaming healthcare-related data into the cloud. It provides a "front door" for [Data Integrators](glossary.md#data-integrator) to send data into the IBM&reg; Cloud, while supporting both batch processing and data streaming workflows. It provides features to initiate and track the movement of a dataset into the IBM Cloud for both Data Integrators and [Data Consumers](glossary.md#data-consumer). 
 
-The key features are:
-- Streaming - all data is streamed
-- Batch support - a collection of health data records can be streamed and processed together
-- Multitenancy - supports segregation of data by tenant and Data Integrator
+**Key features**
```

Comment:
It looks like you've been making lines bold as a way to create headers. In Markdown, this should be done by adding '#''s to the beginning of the line. Each additional '#' increases the heading level with one '#' being the biggest.